packages/project-build/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "@webstudio-is/tsconfig/base.json",
-  "include": ["src"],
+  "include": ["src", "../../@types/**/*.d.ts"],
```

Comment:
cc @TrySound @Andarist Im not sure this will work

Reply:
`@types/css-tree` is not exposing default generated instances types of things like Lexer.

Up until this point `apps/builder` referenced these types for CSS Tree (because of the reason above). In short I didn't change anything so it shouldn't break

Reply:
Quite frankly, those kinds of options always confuse me - it gets really tricky when u mix similar-purpose options. I can't say with confidence that I know what kind of outcome this might have for your `.d.ts` generation and stuff. However, this was already configured like this for some packages so I guess that this might be roughly OK. I have a feeling though that this should use `typeRoots` and not `include` - but if it works... I wouldn't bother changing this right now.

Reply:
> Quite frankly, those kinds of options always confuse me - it gets really tricky when u mix similar-purpose options

I agree with you! When `typeRoots` for some reason these are not picked up though. I created an issue about revisiting the types for css-tree https://github.com/webstudio-is/webstudio-builder/issues/1663